### PR TITLE
Styled normal and active editor tabs

### PR DESCRIPTION
**Pull request by felipemendes**
Styled Posts and Pages editor tabs with the recommended colors

### CODE REVIEW

**Comment by danieltj27** [comment]
Thanks for submitting this, I'll take a proper look soon.

File: darkmode.scss [hunk 1]
```diff
@@ -1786,5 +1786,19 @@ body {
 
 	}
 
-}
+	.wp-switch-editor {
```
Comment (danieltj27) [inline darkmode.scss]: Where is this class used? I originally thought it was for the TinyMCE editor but it doesn't seem to have had any effect.

File: darkmode.scss [hunk 2]
```diff
@@ -1786,5 +1786,19 @@ body {
...
+		background: $dark-silver;
+		color: $white;
+		border-color: $dark-grey;
+		&:active {
+			background-color: $dark-grey;
+		}
+	}
+	.html-active .switch-html, .tmce-active .switch-tmce {
```
Comment (danieltj27) [inline darkmode.scss]: See above comment - where is this used?

Comment (felipemendes) [reply darkmode.scss]: Hey. That's ok. The changes should really be applied at TinyMCE, on tabs Visual and Text. For some reason, on v1.7, the dark mode it's not been applied anymore locally. When I upgrade it to v1.8 it works fine. If I copy these scss styles it works fine changing the tabs.

Comment (danieltj27) [reply darkmode.scss]: Okay, I'll take a look and see if it works on my end soon. I'm also inclined as per issue #77 to style the rest of the editor too. Would be worth merging this change in with the editor styling changes.

**Comment by danieltj27** [comment]
I'll close this due to the inactivity but feel free to continue discussion in #77 and submit another PR relating to this new ticket that's been created. Everything for TinyMCE will be tracked their and new PRs. Thank you :)

**Pull request closed without merging**